client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -76,7 +84,8 @@ export class ProductPurchaseFeaturesList extends Component {
 				<GoogleAnalyticsStats selectedSite={ selectedSite } />
 				<GoogleMyBusiness selectedSite={ selectedSite } />
 				<AdvertisingRemoved isBusinessPlan selectedSite={ selectedSite } />
-				<CustomizeTheme selectedSite={ selectedSite } />
+				<QueryBlogStickers blogId={ selectedSite.ID } />
```

Comment:
In the connect method we can expose `selectedSiteId` and then this line becomes:
```
const { selectedSiteId, ... } = this.props;
//...
<QueryBlogStickers blogId={ selectedSiteId } />
```